Automattic/wp-calypso
# Add/checklist mobile nudge

This makes the components from the /me/get-apps/ page into a block and then uses the block after the WPcom site setup checklist is complete.

Fixes #34381.

**Before** | **After**
------------ | -------------
![Screen Shot 2019-07-08 at 1 31 59 PM](https://user-images.githubusercontent.com/942359/60830330-f2a27980-a184-11e9-8a45-5cff517a495f.png) | ![Screen Shot 2019-07-08 at 1 29 39 PM](https://user-images.githubusercontent.com/942359/60830359-051cb300-a185-11e9-9d40-ae640100e82b.png)

**To Test:**

- on a new site or a site with incomplete checklist items visit `/checklist/{yoursite}`
- verify that the app nudge is not displayed while checklist items are incomplete
- complete all checklist items (with the exception of email verification, can be done by clicking the circle beside the item)
- when the checklist congratulations message is shown, the app nudge should be shown
- verify that the app nudge links and text message field work as expected
- visit `/me/get-apps`
- verify that the apps block works as expected and is unchanged
